src/app_charts/prometheus/prometheus-cloud.values.yaml

```diff
@@ -56,6 +58,9 @@ coreDns:
 kubelet:
   enabled: false
 
+# Throws a invalid namespace "kube-system" error during deployment
```

Comment:
typo 'hrows an invalid ', also in prometheus-robot.values.yaml

Reply:
Can't see it ;)

Reply:
ok now I got it :)